2/src/part2.c

```diff
@@ -0,0 +1,74 @@
+// Link: https://adventofcode.com/2022/day/2
+
+#include <stdio.h>
+
+enum {
+    ROCK = 1,
+    PAPER,
+    SCISSORS
+};
+
+int decode_move(char move) {
+    switch (move) {
+        case 'A':  // Rock
+            return ROCK;
+        case 'B':  // Paper
+            return PAPER;
+        case 'C':  // Scissors
+            return SCISSORS;
+        default:
+            return 0;
```

Comment:
```suggestion
            return INVALID;
```
add to the enum list
    INVALID = -1